pkg/logging/logger.go

```diff
@@ -274,6 +278,22 @@ func (*logrusEntryWrapper) IsTracing() bool {
 	return defaultLogger.IsLevelEnabled(logrus.TraceLevel)
 }
 
+func (*logrusEntryWrapper) IsDebugging() bool {
+	return defaultLogger.IsLevelEnabled(logrus.DebugLevel)
+}
```

Comment:
All these `Is*` methods seem wrong, they should look something like
```go
func (l *logrusEntryWrapper) IsDebugging() bool {
	return l.e.Logger.IsLevelEnabled(logrus.DebugLevel)
}
```

Currently they may look at the wrong Logger.

@nopcoder: if this is true, separate issue for this perhaps?